ctms/app.py

```diff
@@ -1,5 +1,6 @@
 """The CTMS application, including middleware and routes."""
 # pylint:disable = too-many-lines
+# pylint:disable = too-many-statements
```

Comment:
ouch! :hurtrealbad:

It may be worth breaking this into more files, but definitely outside of this PR.

Reply:
https://github.com/mozilla-it/ctms-api/issues/360

Constructed that issue to keep track of this, thanks for the call out.